demo-projects/blog/package.json

```diff
@@ -43,7 +43,7 @@
     "graphql-tag": "^2.10.3",
     "isomorphic-unfetch": "^3.0.0",
     "next": "^9.4.4",
-    "node-fetch": "^2.3.0",
+    "node-fetch": "^2.6.0",
```

Comment:
Can we break out these version bumps into a separate PR? There's a lot going on in this one already.